proposals/dao/fip_63.ts

```diff
@@ -0,0 +1,108 @@
+import hre, { ethers } from 'hardhat';
+import chai, { expect } from 'chai';
+import CBN from 'chai-bn';
+import { DeployUpgradeFunc, SetupUpgradeFunc, TeardownUpgradeFunc, ValidateUpgradeFunc } from '@custom-types/types';
+
+chai.use(CBN(ethers.BigNumber));
+
+const eth = ethers.constants.WeiPerEther;
+const toBN = ethers.BigNumber.from;
+
+/*
+FIP-63
+DEPLOY ACTIONS:
+
+1. Deploy lusdPSM
+  -- target of LUSD PSM will be bammDeposit
+  -- reserve threshold will be 10m lusd
+  -- mint fee 50 basis points
+  -- redeem fee 50 basis points
+
+*/
+
+const decimalsNormalizer = 0;
+const doInvert = false;
+
+const mintFeeBasisPoints = 50;
+const redeemFeeBasisPoints = 50;
+const reservesThreshold = toBN(10_000_000).mul(eth);
+const feiMintLimitPerSecond = ethers.utils.parseEther('10000');
+const lusdPSMBufferCap = ethers.utils.parseEther('10000000');
+
+const incentiveAmount = 0;
+
+const lusdDripAmount = ethers.utils.parseEther('5000000');
+const dripDuration = 1800;
+
+export const deploy: DeployUpgradeFunc = async (deployAddress, addresses, logging = false) => {
+  const { bammDeposit, core, chainlinkLUSDOracleWrapper, lusd } = addresses;
+
+  if (!core) {
+    throw new Error('An environment variable contract address is not set');
+  }
+
+  // 1. Deploy eth PSM
+  const lusdPSM = await (
+    await ethers.getContractFactory('MintRedeemPausePSM')
+  ).deploy(
+    {
+      coreAddress: core,
+      oracleAddress: chainlinkLUSDOracleWrapper,
+      backupOracle: chainlinkLUSDOracleWrapper,
```

Comment:
leave backup address 0

Reply:
in previous PSM's, the backup address is the same as the main one.

Reply:
```
        if (_backupOracle != address(0) && _backupOracle != _oracle) {
            _setBackupOracle(_backupOracle);
        }
```
won't do anything. Fine either way its < 100 gas to do the comparison

Reply:
gas optimization is a good reason to not lindy this one